RELEASES.md

```diff
@@ -722,7 +722,6 @@ Compatibility Notes
 
 - [Only support Android NDK 25 or newer](https://blog.rust-lang.org/2023/01/09/android-ndk-update-r25.html)
 - [Add `SEMICOLON_IN_EXPRESSIONS_FROM_MACROS` to future-incompat report](https://github.com/rust-lang/rust/pull/103418/)
-- [Only specify `--target` by default for `-Zgcc-ld=lld` on wasm](https://github.com/rust-lang/rust/pull/101792/)
```

Comment:
Not sure how bugfix for an unstable feature got into release notes.

Reply:
Unstable options are sometimes mentioned in the release notes, but it's quite rare indeed. Maybe we should leave it as-is?

The PR didn't appear in the blog post for the 1.68.0 release at least.

Reply:
I don't have a strong opinion, just removed it to avoid removed gcc-ld being mentioned in the repo.

Reply:
There's likely no pressing need to erase the option from history, so let's make our lives easier by not needing to ask anyone how it's supposed to work 😅 -- and drop the changes to this file.